changelogs/fragments/3343-redfish_utils-addUser-userId.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
So this is a feature (and not a bugfix)?

Reply:
Oh, no it is a bugfix. I'll change it.

Reply:
I wouldn't call this a bugfix; this is a workaround to a bug on a system. Most systems I've worked with do not require the "Id" property to be sent to the service. I would also not typically expect this to be normal behavior since "Id" is a service-managed property. While it's legal for a client to request the Id property, it's very odd for a service to require it to be specified.